docs/ism/policies.md

```diff
@@ -8,13 +8,248 @@ has_children: false
+
+Parameter | Description | Type | Required
+:--- | :--- |:--- |:--- |
+`name` |  The name of the state. | `string` | Yes
+`actions` | The actions to execute after entering a state. For more information, see [Actions](#actions). | `nested list of objects` | Yes
+`transitions` | The next states and conditions required to transition to those states, if no transitions are specified then the policy assumes that it's complete and can now stop managing the index. For more information, see [Transitions](#transitions). | `nested list of objects` | Yes
+
+---
+
+## Actions
+
+Actions define the steps that'll be sequentially executed on entering a specific state.
+
+This table lists the parameters you can define for an action:
+
+Parameter | Description | Type | Required | Default
+:--- | :--- |:--- |:--- |
+`timeout` |  The timeout period for the action, accepts time units for minutes, hours, and days. | `time unit` | No | specific to action
+`retry` | The retry configuration for the action. | `object` | No | specific to action
+`count` | The number of retry counts. | `number` | No | specific to action
+`backoff` | The backoff policy type to use when retrying. | `string` | No | exponential
+`delay` | The time to wait between retries, accepts time units for minutes, hours, and days. | `time unit` | No | specific to action
+
+All actions have common `timeout`, `retry`, and `backoff` settings. If they are not specified, appropriate defaults per action are used.
+
+Here's a sample action that has a timeout period of one hour, that is retried three times with the exponential backup policy, with a delay of five seconds between each retry:
+
+```json
+{
+    "timeout": "1h",
+    "retry": {
+        "count": 3,
+        "backoff": "exponential",
+        "delay": "5s"
+    }
+}
+```
+
+
+The following actions are supported:
+
+- [forcemerge](#forcemerge)
+- [read_only](#read_only)
+- [read_write](#read_write)
+- [replica_count](#replica_count)
+- [close](#close)
+- [open](#open)
+- [delete](#delete)
+- [snapshot](#snapshot)
+- [rollover](#rollover)
+
+### forcemerge
+
+Reduces the number of Lucene segments by merging the indices.
+It attempts to set the index to a read-only state before starting the merging process.
+
+Parameter | Description | Type | Required | Default
+:--- | :--- |:--- |:--- |
+`max_num_segments` | The number of segments to reduce the shard to. | `number` | Yes | Null
+
+```json
+{
+    "forcemerge": {
+        "max_num_segments": 1
+    }
+}
+```
+
+### read_only
+
+Sets a managed index to be read only.
+
+```json
+{
+    "read_only": {}
+}
+```
+
+### read_write
+
+Sets a managed index to be writeable.
+
+```json
+{
+    "read_write": {}
+}
+```
+
+### replica_count
+
+Changes the number of replicas of the managed index.
+
+Parameter | Description | Type | Required
+:--- | :--- |:--- |:--- |
+`number_of_replicas` | It defines the number of replicas to assign to an index. | `number` | Yes
```

Comment:
I think you can ditch the intro to this, given that it's duplicative with the description in the table.